src/index.js

```diff
@@ -1,20 +1,54 @@
 import path from 'path';
 
-export default function({types: t}) {
+export default function({ types: t }) {
+  function getValue(src, name, key) {
+    const basePath = path.resolve(src);
+    const pkg = require(path.join(basePath, '..', name));
+
+    return pkg[key];
+  }
+
+  function isManifest(node) {
+    return t.isLiteral(node) && /package\.json$/.test(node.value);
+  }
+
   return {
     visitor: {
-      MemberExpression: function MemberExpression(treePath) {
-        const node = treePath.node;
-        if (t.isCallExpression(node.object) &&
-            t.isIdentifier(node.object.callee, { name: 'require' }) &&
-            t.isLiteral(node.object.arguments[0]) &&
-            node.object.arguments[0].value.match(/package\.json$/)) {
-          let srcPath = path.resolve(this.file.opts.filename);
-          let pkg = require(path.join(srcPath, '..', node.object.arguments[0].value));
-          let value = pkg[node.property.name];
-          treePath.replaceWith(t.expressionStatement(t.valueToNode(value)));
+      ImportDeclaration: function importDeclaration(path) {
```

Comment:
let's use `treePath` here over `path`, otherwise we get implicit conflicts with the `path` module imported at the top.